raiden/api/python.py

```diff
@@ -193,25 +193,25 @@ def get_transfer_from_task(
 
 
 def transfer_tasks_view(
-    transfer_tasks: Dict[SecretHash, TransferTask],
+    payment_mapping: Dict[Address, PaymentMappingState],
     token_address: TokenAddress = None,
     channel_id: ChannelID = None,
 ) -> List[Dict[str, Any]]:
     view = list()
 
-    for secrethash, transfer_task in transfer_tasks.items():
-        transfer, role = get_transfer_from_task(secrethash, transfer_task)
-
-        if transfer is None:
-            continue
-        if token_address is not None:
-            if transfer.token != token_address:
+    for node_address in payment_mapping.keys():
+        for secrethash, transfer_task in payment_mapping[node_address].items():
```

Comment:
`for k in d`—when `d` is a dictionary—already assigns key values to `k`, so you do not need to use `.keys()` in python 3.

moreover, you can call `.values()` instead, if you're not going to use the keys for anything (as it seems to be the case here).